modules/UI/videolayout/VideoContainer.js

```diff
@@ -99,21 +99,28 @@ function computeCameraVideoSize( // eslint-disable-line max-params
     }
 
     const aspectRatio = videoWidth / videoHeight;
+    const videoSpaceRatio = videoSpaceWidth / videoSpaceHeight;
+
+    if (videoSpaceRatio === aspectRatio) {
```

Comment:
Moving this here, that changes the rest of the logic, isn't it? Before your change that was executed only in the case of both `videoLayoutFit` where now it is always?

Reply:
It's true that this check is happening more often now, but I also don't think it actually changes the logic, unless I'm missing something (which is possible; it's been a long day...).

I think it's fine to short-circuit here, and that this should always produce the correct result regardless of the chosen fit method.  By definition, all of the expressions below evaluate to `[ videoSpaceWidth, videoSpaceHeight ]` if `videoSpaceRatio` and `aspectRatio` are the same, regardless of the value of `VIDEO_LAYOUT_FIT`, right?

Reply:
@saghul or @muscat1 can you take a look :) as I'm not very familiar with this part :)

Reply:
Oh, looking a little more closely, I guess one exception is the default case, which would be different with this change (since this would no longer keep the video at its original size. I can change it back if that's preferable.

Reply:
(done).  

Happy to reconsider that change, or to make others.  I can also squash these commits once everything looks OK.